images_config/ubuntu-auto.seed

```diff
@@ -45,4 +45,5 @@ d-i partman/confirm_nooverwrite boolean true
 # If we're booting using the backported Vivid kernel, install it too.
 d-i	base-installer/kernel/altmeta	string lts-xenial
 
-
+#use cob kitchen server as apt cacher
+d-i mirror/http/proxy string http://10.0.1.1:3142/
```

Comment:
this means we can only install pc if we are within the cob-kitchen-server network. please provide two options in grub for installation with and without apt-cacher.